ios/Cartfile

```diff
@@ -1,2 +1,4 @@
 github "stephencelis/SQLite.swift" "swift-2.3"
-github "mozilla-magnet/magnet-scanner-ios" ~> 0.2.3
+github "arcturus/magnet-scanner-ios" "geo-scanner"
+github "Alamofire/Alamofire" ~> 3.5.0
+github "SwiftyJSON/SwiftyJSON" "swift2"
```

Comment:
These are dependencies of `magnet-scanner-ios` no? Why do they have to be redefined in the host project?